sdk/eventhub/event-hubs/samples/typescript/src/iothubConnectionString.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  This sample demonstrates how to convert an Iot Hubs connection string to
+  an Event Hubs-compatible connection string that can be used with the EventHubConsumerClient
+  to process events from an Iot Hub.
+*/
+
+import * as crypto from "crypto";
+import { Buffer } from "buffer";
+import {
+  AmqpError,
+  Connection,
+  ReceiverEvents,
+  isAmqpError as rheaIsAmqpError,
+  parseConnectionString
+} from "rhea-promise";
+import { EventHubConsumerClient, earliestEventPosition } from "@azure/event-hubs";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+/**
+ * Type guard for AmqpError.
+ * @param err An unknown error.
+ */
+function isAmqpError(err: any): err is AmqpError {
+  return rheaIsAmqpError(err);
+}
+
+const consumerGroup = process.env["CONSUMER_GROUP_NAME"] || "";
+
+// This code is modified from https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#security-tokens.
+function generateSasToken(
+  resourceUri: string,
+  signingKey: string,
+  policyName: string,
+  expiresInMins: number
+): string {
+  resourceUri = encodeURIComponent(resourceUri);
+
+  // Set expiration in seconds.
```

Comment:
You can probably delete this comment if you rename the variable below

Reply:
Updated.